#!/usr/bin/make -f
#Generated by AXIOME2

all: mapping/copy/metadata_mapping.txt source/fasta0/SBSD.fasta.gz source/fasta1/AS2.fasta.gz source/fasta2/AS3.fasta.gz source/fasta3/BH2.fasta.gz source/fasta4/BARB.fasta.gz source/fasta5/RRH.fasta.gz source/fasta6/HY1.fasta.gz source/fasta7/LI1.fasta.gz source/fasta8/LI2.fasta.gz source/fasta9/LI3.fasta.gz source/fasta10/GP.fasta.gz source/fasta11/RP.fasta.gz source/fasta12/RPSD.fasta.gz source/fasta13/SB.fasta.gz source/fasta14/BB1.fasta.gz source/fasta15/BB2.fasta.gz source/fasta16/PB1.fasta.gz source/fasta17/PB2.fasta.gz source/fasta18/bat.fasta.gz source/fasta19/FF.fasta.gz source/fasta20/RHSD.fasta.gz source/fasta21/HH.fasta.gz source/fasta22/RH.fasta.gz source/fasta23/SPIM.fasta.gz source/fasta24/MAR.fasta.gz source/fasta25/EAC.fasta.gz source/fasta26/BAZ.fasta.gz source/fasta27/FL.fasta.gz source/fasta28/GOR.fasta.gz source/fasta29/GORSD.fasta.gz source/fasta30/CHIMP1.fasta.gz source/fasta31/CHIMP12.fasta.gz source/fasta32/orang1.fasta.gz source/fasta33/orang2.fasta.gz source/fasta34/ML.fasta.gz source/fasta35/RT.fasta.gz source/fasta36/Saki.fasta.gz source/fasta37/AFEL.fasta.gz source/fasta38/AFEL2.fasta.gz source/fasta39/AFEL3.fasta.gz source/fasta40/SQ.fasta.gz source/fasta41/arma.fasta.gz source/fasta42/KO1.fasta.gz source/fasta43/ECH.fasta.gz merge_sources/concatenate/seq.fasta merge_sources/concatenate/chimeric_otus.fasta cluster/UPARSE/seq.unique.fasta cluster/UPARSE/seq.unique.sorted.fasta cluster/UPARSE/otus_original_label.fasta cluster/UPARSE/rep_set.fasta cluster/UPARSE/map.uc cluster/UPARSE/seq_otus.txt rep_set/UPARSE/rep_set.fasta classification/qiime_rdp/rep_set_tax_assignments.txt alignment/qiime_pynast/rep_set_aligned.fasta alignment/qiime_pynast/rep_set_aligned_pfiltered.fasta phylogeny/qiime_fasttree/rep_set.tre create_otu_table/qiime0/otu_table.biom rarefy_otu_table/qiime_even0/otu_table_summary.txt rarefy_otu_table/qiime_even0/otu_table_auto.biom convert_otu_table/full_biom_to_tab0/otu_table.tab convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab misc_analysis/biom_summarize-table0/table_summary.txt misc_analysis/mesas_representative_sequences0/otu_table_auto_with_seq.tab misc_analysis/mesas_representative_sequences0/otu_table_with_seq.tab misc_analysis/qiime_alpha_rarefaction0/alpha_rarefaction_plots/rarefaction_plots.html rarefied_analysis/mesas_pcoa0/pcoa-bray.pdf rarefied_analysis/mesas_indicator-species0/ind_species_raw.txt rarefied_analysis/mesas_indicator-species0/indicator_species.html rarefied_analysis/mesas_mrpp0/mrpp.txt rarefied_analysis/qiime_unifrac-pcoa0/otu_table_auto_L7.txt rarefied_analysis/qiime_unifrac-pcoa0/weighted_unifrac_emperor_pcoa_plot/index.html rarefied_analysis/qiime_unifrac-pcoa0/unweighted_unifrac_emperor_pcoa_plot/index.html rarefied_analysis/qiime_2d-pcoa0/distance/bray_curtis_otu_table_auto.txt rarefied_analysis/qiime_2d-pcoa0/pcoa/pcoa_bray_curtis_otu_table_auto.txt rarefied_analysis/qiime_2d-pcoa0/pcoa_plots/_2D_PCoA_plots.html rarefied_analysis/qiime_compare-sample-groups0/distance/bray_curtis_otu_table_auto.txt rarefied_analysis/qiime_compare-sample-groups0/mrpp_results.txt rarefied_analysis/qiime_alpha-comparison0/alpha_rarefaction_plots/rarefaction_plots.html rarefied_analysis/mesas_nmds0/nmds.pdf Feb26.zip

mapping/copy/metadata_mapping.txt: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammal_Metadata_Ashley.tsv
	mkdir -p mapping/copy/
	axiome utility metadata_file_check -m /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammal_Metadata_Ashley.tsv && cp /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammal_Metadata_Ashley.tsv mapping/copy/metadata_mapping.txt

source/fasta0/SBSD.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta0/
	awk -v MATCHING=SBSD 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=SBSD 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta0/SBSD.fasta && gzip -f source/fasta0/SBSD.fasta

source/fasta1/AS2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta1/
	awk -v MATCHING=AS2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=AS2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta1/AS2.fasta && gzip -f source/fasta1/AS2.fasta

source/fasta2/AS3.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta2/
	awk -v MATCHING=AS3 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=AS3 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta2/AS3.fasta && gzip -f source/fasta2/AS3.fasta

source/fasta3/BH2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta3/
	awk -v MATCHING=BH2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=BH2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta3/BH2.fasta && gzip -f source/fasta3/BH2.fasta

source/fasta4/BARB.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta4/
	awk -v MATCHING=BARB 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=BARB 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta4/BARB.fasta && gzip -f source/fasta4/BARB.fasta

source/fasta5/RRH.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta5/
	awk -v MATCHING=RRH 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=RRH 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta5/RRH.fasta && gzip -f source/fasta5/RRH.fasta

source/fasta6/HY1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta6/
	awk -v MATCHING=HY1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=HY1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta6/HY1.fasta && gzip -f source/fasta6/HY1.fasta

source/fasta7/LI1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta7/
	awk -v MATCHING=LI1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=LI1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta7/LI1.fasta && gzip -f source/fasta7/LI1.fasta

source/fasta8/LI2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta8/
	awk -v MATCHING=LI2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=LI2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta8/LI2.fasta && gzip -f source/fasta8/LI2.fasta

source/fasta9/LI3.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta9/
	awk -v MATCHING=LI3 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=LI3 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta9/LI3.fasta && gzip -f source/fasta9/LI3.fasta

source/fasta10/GP.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta10/
	awk -v MATCHING=GP 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=GP 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta10/GP.fasta && gzip -f source/fasta10/GP.fasta

source/fasta11/RP.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta11/
	awk -v MATCHING=RP 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=RP 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta11/RP.fasta && gzip -f source/fasta11/RP.fasta

source/fasta12/RPSD.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta12/
	awk -v MATCHING=RPSD 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=RPSD 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta12/RPSD.fasta && gzip -f source/fasta12/RPSD.fasta

source/fasta13/SB.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta13/
	awk -v MATCHING=SB 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=SB 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta13/SB.fasta && gzip -f source/fasta13/SB.fasta

source/fasta14/BB1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta14/
	awk -v MATCHING=BB1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=BB1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta14/BB1.fasta && gzip -f source/fasta14/BB1.fasta

source/fasta15/BB2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta15/
	awk -v MATCHING=BB2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=BB2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta15/BB2.fasta && gzip -f source/fasta15/BB2.fasta

source/fasta16/PB1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta16/
	awk -v MATCHING=PB1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=PB1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta16/PB1.fasta && gzip -f source/fasta16/PB1.fasta

source/fasta17/PB2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta17/
	awk -v MATCHING=PB2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=PB2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta17/PB2.fasta && gzip -f source/fasta17/PB2.fasta

source/fasta18/bat.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta18/
	awk -v MATCHING=bat 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=bat 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta18/bat.fasta && gzip -f source/fasta18/bat.fasta

source/fasta19/FF.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta19/
	awk -v MATCHING=FF 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=FF 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta19/FF.fasta && gzip -f source/fasta19/FF.fasta

source/fasta20/RHSD.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta20/
	awk -v MATCHING=RHSD 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=RHSD 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta20/RHSD.fasta && gzip -f source/fasta20/RHSD.fasta

source/fasta21/HH.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta21/
	awk -v MATCHING=HH 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=HH 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta21/HH.fasta && gzip -f source/fasta21/HH.fasta

source/fasta22/RH.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta22/
	awk -v MATCHING=RH 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=RH 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta22/RH.fasta && gzip -f source/fasta22/RH.fasta

source/fasta23/SPIM.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta23/
	awk -v MATCHING=SPIM 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=SPIM 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta23/SPIM.fasta && gzip -f source/fasta23/SPIM.fasta

source/fasta24/MAR.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta24/
	awk -v MATCHING=MAR 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=MAR 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta24/MAR.fasta && gzip -f source/fasta24/MAR.fasta

source/fasta25/EAC.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta25/
	awk -v MATCHING=EAC 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=EAC 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta25/EAC.fasta && gzip -f source/fasta25/EAC.fasta

source/fasta26/BAZ.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta26/
	awk -v MATCHING=BAZ 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=BAZ 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta26/BAZ.fasta && gzip -f source/fasta26/BAZ.fasta

source/fasta27/FL.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta27/
	awk -v MATCHING=FL 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=FL 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta27/FL.fasta && gzip -f source/fasta27/FL.fasta

source/fasta28/GOR.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta28/
	awk -v MATCHING=GOR 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=GOR 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta28/GOR.fasta && gzip -f source/fasta28/GOR.fasta

source/fasta29/GORSD.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta29/
	awk -v MATCHING=GORSD 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=GORSD 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta29/GORSD.fasta && gzip -f source/fasta29/GORSD.fasta

source/fasta30/CHIMP1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta30/
	awk -v MATCHING=CHIMP1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=CHIMP1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta30/CHIMP1.fasta && gzip -f source/fasta30/CHIMP1.fasta

source/fasta31/CHIMP12.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta31/
	awk -v MATCHING=CHIMP12 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=CHIMP12 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta31/CHIMP12.fasta && gzip -f source/fasta31/CHIMP12.fasta

source/fasta32/orang1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta32/
	awk -v MATCHING=orang1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=orang1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta32/orang1.fasta && gzip -f source/fasta32/orang1.fasta

source/fasta33/orang2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta33/
	awk -v MATCHING=orang2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=orang2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta33/orang2.fasta && gzip -f source/fasta33/orang2.fasta

source/fasta34/ML.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta34/
	awk -v MATCHING=ML 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=ML 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta34/ML.fasta && gzip -f source/fasta34/ML.fasta

source/fasta35/RT.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta35/
	awk -v MATCHING=RT 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=RT 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta35/RT.fasta && gzip -f source/fasta35/RT.fasta

source/fasta36/Saki.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta36/
	awk -v MATCHING=Saki 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=Saki 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta36/Saki.fasta && gzip -f source/fasta36/Saki.fasta

source/fasta37/AFEL.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta37/
	awk -v MATCHING=AFEL 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=AFEL 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta37/AFEL.fasta && gzip -f source/fasta37/AFEL.fasta

source/fasta38/AFEL2.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta38/
	awk -v MATCHING=AFEL2 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=AFEL2 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta38/AFEL2.fasta && gzip -f source/fasta38/AFEL2.fasta

source/fasta39/AFEL3.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta39/
	awk -v MATCHING=AFEL3 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=AFEL3 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta39/AFEL3.fasta && gzip -f source/fasta39/AFEL3.fasta

source/fasta40/SQ.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta40/
	awk -v MATCHING=SQ 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=SQ 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta40/SQ.fasta && gzip -f source/fasta40/SQ.fasta

source/fasta41/arma.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta41/
	awk -v MATCHING=arma 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=arma 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta41/arma.fasta && gzip -f source/fasta41/arma.fasta

source/fasta42/KO1.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta42/
	awk -v MATCHING=KO1 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=KO1 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta42/KO1.fasta && gzip -f source/fasta42/KO1.fasta

source/fasta43/ECH.fasta.gz: /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta
	mkdir -p source/fasta43/
	awk -v MATCHING=ECH 'BEGIN{ORS=""; out=0;} {if (!/^>/ && out) {print;} if (/^>/ && ($$0~MATCHING)) {if (out==1) {print "\n"}; print; print "\n"; out=1;} if (/^>/ && ($$0!~MATCHING)) {if (out==1) {print "\n";} out=0;}}END{if (out==1) {print "\n"}}' /Winnebago/aaross/Feb26_Ashley_Mammals_SamplesRemoved/Mammals_ultraclean_no_chimeras_17504.fasta | awk -v SAMPLEID=ECH 'BEGIN{count=0}{if (/^>/){print ">" SAMPLEID "_" count; count+=1}else{print}}' > source/fasta43/ECH.fasta && gzip -f source/fasta43/ECH.fasta

merge_sources/concatenate/seq.fasta merge_sources/concatenate/chimeric_otus.fasta: source/fasta0/SBSD.fasta.gz source/fasta1/AS2.fasta.gz source/fasta2/AS3.fasta.gz source/fasta3/BH2.fasta.gz source/fasta4/BARB.fasta.gz source/fasta5/RRH.fasta.gz source/fasta6/HY1.fasta.gz source/fasta7/LI1.fasta.gz source/fasta8/LI2.fasta.gz source/fasta9/LI3.fasta.gz source/fasta10/GP.fasta.gz source/fasta11/RP.fasta.gz source/fasta12/RPSD.fasta.gz source/fasta13/SB.fasta.gz source/fasta14/BB1.fasta.gz source/fasta15/BB2.fasta.gz source/fasta16/PB1.fasta.gz source/fasta17/PB2.fasta.gz source/fasta18/bat.fasta.gz source/fasta19/FF.fasta.gz source/fasta20/RHSD.fasta.gz source/fasta21/HH.fasta.gz source/fasta22/RH.fasta.gz source/fasta23/SPIM.fasta.gz source/fasta24/MAR.fasta.gz source/fasta25/EAC.fasta.gz source/fasta26/BAZ.fasta.gz source/fasta27/FL.fasta.gz source/fasta28/GOR.fasta.gz source/fasta29/GORSD.fasta.gz source/fasta30/CHIMP1.fasta.gz source/fasta31/CHIMP12.fasta.gz source/fasta32/orang1.fasta.gz source/fasta33/orang2.fasta.gz source/fasta34/ML.fasta.gz source/fasta35/RT.fasta.gz source/fasta36/Saki.fasta.gz source/fasta37/AFEL.fasta.gz source/fasta38/AFEL2.fasta.gz source/fasta39/AFEL3.fasta.gz source/fasta40/SQ.fasta.gz source/fasta41/arma.fasta.gz source/fasta42/KO1.fasta.gz source/fasta43/ECH.fasta.gz
	mkdir -p merge_sources/concatenate/
	zcat source/fasta0/SBSD.fasta.gz source/fasta1/AS2.fasta.gz source/fasta2/AS3.fasta.gz source/fasta3/BH2.fasta.gz source/fasta4/BARB.fasta.gz source/fasta5/RRH.fasta.gz source/fasta6/HY1.fasta.gz source/fasta7/LI1.fasta.gz source/fasta8/LI2.fasta.gz source/fasta9/LI3.fasta.gz source/fasta10/GP.fasta.gz source/fasta11/RP.fasta.gz source/fasta12/RPSD.fasta.gz source/fasta13/SB.fasta.gz source/fasta14/BB1.fasta.gz source/fasta15/BB2.fasta.gz source/fasta16/PB1.fasta.gz source/fasta17/PB2.fasta.gz source/fasta18/bat.fasta.gz source/fasta19/FF.fasta.gz source/fasta20/RHSD.fasta.gz source/fasta21/HH.fasta.gz source/fasta22/RH.fasta.gz source/fasta23/SPIM.fasta.gz source/fasta24/MAR.fasta.gz source/fasta25/EAC.fasta.gz source/fasta26/BAZ.fasta.gz source/fasta27/FL.fasta.gz source/fasta28/GOR.fasta.gz source/fasta29/GORSD.fasta.gz source/fasta30/CHIMP1.fasta.gz source/fasta31/CHIMP12.fasta.gz source/fasta32/orang1.fasta.gz source/fasta33/orang2.fasta.gz source/fasta34/ML.fasta.gz source/fasta35/RT.fasta.gz source/fasta36/Saki.fasta.gz source/fasta37/AFEL.fasta.gz source/fasta38/AFEL2.fasta.gz source/fasta39/AFEL3.fasta.gz source/fasta40/SQ.fasta.gz source/fasta41/arma.fasta.gz source/fasta42/KO1.fasta.gz source/fasta43/ECH.fasta.gz > merge_sources/concatenate/seq.fasta && echo '' > merge_sources/concatenate/chimeric_otus.fasta

cluster/UPARSE/seq.unique.fasta: merge_sources/concatenate/seq.fasta
	mkdir -p cluster/UPARSE/
	mesas-dereplicate -i merge_sources/concatenate/seq.fasta -o cluster/UPARSE/seq.unique.fasta -n 2

cluster/UPARSE/seq.unique.sorted.fasta: cluster/UPARSE/seq.unique.fasta
	mkdir -p cluster/UPARSE/
	usearch -sortbysize cluster/UPARSE/seq.unique.fasta -output cluster/UPARSE/seq.unique.sorted.fasta -minsize 2

cluster/UPARSE/otus_original_label.fasta: cluster/UPARSE/seq.unique.sorted.fasta
	mkdir -p cluster/UPARSE/
	usearch -cluster_otus cluster/UPARSE/seq.unique.sorted.fasta -otus cluster/UPARSE/otus_original_label.fasta -otu_radius_pct `echo "(1.0-0.97)*100" | bc`

cluster/UPARSE/rep_set.fasta: cluster/UPARSE/otus_original_label.fasta
	mkdir -p cluster/UPARSE/
	awk 'BEGIN{count=0;}{if ($$0~/>/){print ">" count; count+=1;} else {print}}' cluster/UPARSE/otus_original_label.fasta > cluster/UPARSE/rep_set.fasta

cluster/UPARSE/map.uc: cluster/UPARSE/rep_set.fasta merge_sources/concatenate/seq.fasta
	mkdir -p cluster/UPARSE/
	usearch -usearch_global merge_sources/concatenate/seq.fasta -db cluster/UPARSE/rep_set.fasta -strand both -id 0.97 -uc cluster/UPARSE/map.uc -threads 4

cluster/UPARSE/seq_otus.txt: cluster/UPARSE/map.uc
	mkdir -p cluster/UPARSE/
	mesas-uc2clust -t 4 cluster/UPARSE/map.uc cluster/UPARSE/seq_otus.txt

rep_set/UPARSE/rep_set.fasta: cluster/UPARSE/rep_set.fasta
	mkdir -p rep_set/UPARSE/
	ln -f cluster/UPARSE/rep_set.fasta rep_set/UPARSE/rep_set.fasta

classification/qiime_rdp/rep_set_tax_assignments.txt: rep_set/UPARSE/rep_set.fasta
	mkdir -p classification/qiime_rdp/
	assign_taxonomy.py -m rdp -i rep_set/UPARSE/rep_set.fasta -o classification/qiime_rdp -c 0.8

alignment/qiime_pynast/rep_set_aligned.fasta: rep_set/UPARSE/rep_set.fasta
	mkdir -p alignment/qiime_pynast/
	align_seqs.py -m pynast -i rep_set/UPARSE/rep_set.fasta -o alignment/qiime_pynast -t /opt/qiime_support/core_set_aligned.fasta.imputed

alignment/qiime_pynast/rep_set_aligned_pfiltered.fasta: alignment/qiime_pynast/rep_set_aligned.fasta
	mkdir -p alignment/qiime_pynast/
	filter_alignment.py -i alignment/qiime_pynast/rep_set_aligned.fasta -o alignment/qiime_pynast -s

phylogeny/qiime_fasttree/rep_set.tre: alignment/qiime_pynast/rep_set_aligned_pfiltered.fasta
	mkdir -p phylogeny/qiime_fasttree/
	make_phylogeny.py -i alignment/qiime_pynast/rep_set_aligned_pfiltered.fasta -t fasttree -o phylogeny/qiime_fasttree/rep_set.tre -l phylogeny/qiime_fasttree/log.txt

create_otu_table/qiime0/otu_table.biom: cluster/UPARSE/seq_otus.txt classification/qiime_rdp/rep_set_tax_assignments.txt merge_sources/concatenate/chimeric_otus.fasta
	mkdir -p create_otu_table/qiime0/
	rm -rf create_otu_table/qiime0/otu_table.biom && make_otu_table.py -i cluster/UPARSE/seq_otus.txt -o create_otu_table/qiime0/otu_table.biom -t classification/qiime_rdp/rep_set_tax_assignments.txt -e merge_sources/concatenate/chimeric_otus.fasta

rarefy_otu_table/qiime_even0/otu_table_summary.txt: create_otu_table/qiime0/otu_table.biom
	mkdir -p rarefy_otu_table/qiime_even0/
	rm -f rarefy_otu_table/qiime_even0/otu_table_summary.txt && biom summarize-table -i create_otu_table/qiime0/otu_table.biom -o rarefy_otu_table/qiime_even0/otu_table_summary.txt

rarefy_otu_table/qiime_even0/otu_table_auto.biom: create_otu_table/qiime0/otu_table.biom rarefy_otu_table/qiime_even0/otu_table_summary.txt
	mkdir -p rarefy_otu_table/qiime_even0/
	single_rarefaction.py -i create_otu_table/qiime0/otu_table.biom -o rarefy_otu_table/qiime_even0/otu_table_auto.biom -d `grep 'Min:' rarefy_otu_table/qiime_even0/otu_table_summary.txt | awk '{print int($$2)}'`

convert_otu_table/full_biom_to_tab0/otu_table.tab: create_otu_table/qiime0/otu_table.biom
	mkdir -p convert_otu_table/full_biom_to_tab0/
	rm -f convert_otu_table/full_biom_to_tab0/otu_table.tab && biom convert --to-tsv -i create_otu_table/qiime0/otu_table.biom --table-type='OTU table' -o convert_otu_table/full_biom_to_tab0/otu_table.tab --header-key=taxonomy --output-metadata-id=Consensus\ Lineage

convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab: rarefy_otu_table/qiime_even0/otu_table_auto.biom
	mkdir -p convert_otu_table/rarefied_biom_to_tab0/
	rm -f convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab && biom convert --to-tsv -i rarefy_otu_table/qiime_even0/otu_table_auto.biom --table-type='OTU table' -o convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab --header-key=taxonomy --output-metadata-id=Consensus\ Lineage

misc_analysis/biom_summarize-table0/table_summary.txt: create_otu_table/qiime0/otu_table.biom
	mkdir -p misc_analysis/biom_summarize-table0/
	rm -f misc_analysis/biom_summarize-table0/table_summary.txt && biom summarize-table -i create_otu_table/qiime0/otu_table.biom -o misc_analysis/biom_summarize-table0/table_summary.txt

misc_analysis/mesas_representative_sequences0/otu_table_auto_with_seq.tab misc_analysis/mesas_representative_sequences0/otu_table_with_seq.tab: convert_otu_table/full_biom_to_tab0/otu_table.tab convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab rep_set/UPARSE/rep_set.fasta
	mkdir -p misc_analysis/mesas_representative_sequences0/
	mesas-otuwithseqs rep_set/UPARSE/rep_set.fasta convert_otu_table/full_biom_to_tab0/otu_table.tab > misc_analysis/mesas_representative_sequences0/otu_table_with_seq.tab && mesas-otuwithseqs rep_set/UPARSE/rep_set.fasta convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab > misc_analysis/mesas_representative_sequences0/otu_table_auto_with_seq.tab

misc_analysis/qiime_alpha_rarefaction0/alpha_rarefaction_plots/rarefaction_plots.html: create_otu_table/qiime0/otu_table.biom mapping/copy/metadata_mapping.txt phylogeny/qiime_fasttree/rep_set.tre
	mkdir -p misc_analysis/qiime_alpha_rarefaction0/
	alpha_rarefaction.py -f -i create_otu_table/qiime0/otu_table.biom -o misc_analysis/qiime_alpha_rarefaction0 -m mapping/copy/metadata_mapping.txt -t phylogeny/qiime_fasttree/rep_set.tre

rarefied_analysis/mesas_pcoa0/pcoa-bray.pdf: convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab mapping/copy/metadata_mapping.txt
	mkdir -p rarefied_analysis/mesas_pcoa0/
	mesas-pcoa -i convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab -m mapping/copy/metadata_mapping.txt -o rarefied_analysis/mesas_pcoa0/pcoa-bray.pdf -d bray

rarefied_analysis/mesas_indicator-species0/ind_species_raw.txt: convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab mapping/copy/metadata_mapping.txt
	mkdir -p rarefied_analysis/mesas_indicator-species0/
	mesas-indicator-species -i convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab -m mapping/copy/metadata_mapping.txt -o rarefied_analysis/mesas_indicator-species0 -p 0.05

rarefied_analysis/mesas_indicator-species0/indicator_species.html: rarefied_analysis/mesas_indicator-species0/ind_species_raw.txt convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab mapping/copy/metadata_mapping.txt
	mkdir -p rarefied_analysis/mesas_indicator-species0/
	mesas-indicator-otu -t convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab -i rarefied_analysis/mesas_indicator-species0/ind_species_raw.txt -m mapping/copy/metadata_mapping.txt -o rarefied_analysis/mesas_indicator-species0

rarefied_analysis/mesas_mrpp0/mrpp.txt: convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab mapping/copy/metadata_mapping.txt
	mkdir -p rarefied_analysis/mesas_mrpp0/
	mesas-mrpp -i convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab -m mapping/copy/metadata_mapping.txt -o rarefied_analysis/mesas_mrpp0 -d bray && mv rarefied_analysis/mesas_mrpp0/mrpp-bray.txt rarefied_analysis/mesas_mrpp0/mrpp.txt

rarefied_analysis/qiime_unifrac-pcoa0/otu_table_auto_L7.txt: rarefy_otu_table/qiime_even0/otu_table_auto.biom
	mkdir -p rarefied_analysis/qiime_unifrac-pcoa0/
	summarize_taxa.py -i rarefy_otu_table/qiime_even0/otu_table_auto.biom -o rarefied_analysis/qiime_unifrac-pcoa0 -L 7

rarefied_analysis/qiime_unifrac-pcoa0/weighted_unifrac_emperor_pcoa_plot/index.html rarefied_analysis/qiime_unifrac-pcoa0/unweighted_unifrac_emperor_pcoa_plot/index.html: rarefy_otu_table/qiime_even0/otu_table_auto.biom mapping/copy/metadata_mapping.txt rarefied_analysis/qiime_unifrac-pcoa0/otu_table_auto_L7.txt phylogeny/qiime_fasttree/rep_set.tre
	mkdir -p rarefied_analysis/qiime_unifrac-pcoa0/
	echo 'make_emperor:taxa_fp rarefied_analysis/qiime_unifrac-pcoa0/otu_table_auto_L7.txt' > rarefied_analysis/qiime_unifrac-pcoa0/params.txt && beta_diversity_through_plots.py -f -i rarefy_otu_table/qiime_even0/otu_table_auto.biom -m mapping/copy/metadata_mapping.txt -t phylogeny/qiime_fasttree/rep_set.tre -o rarefied_analysis/qiime_unifrac-pcoa0 -p rarefied_analysis/qiime_unifrac-pcoa0/params.txt

rarefied_analysis/qiime_2d-pcoa0/distance/bray_curtis_otu_table_auto.txt: rarefy_otu_table/qiime_even0/otu_table_auto.biom phylogeny/qiime_fasttree/rep_set.tre
	mkdir -p rarefied_analysis/qiime_2d-pcoa0/
	beta_diversity.py -i rarefy_otu_table/qiime_even0/otu_table_auto.biom -o rarefied_analysis/qiime_2d-pcoa0/distance/ -m bray_curtis -t phylogeny/qiime_fasttree/rep_set.tre

rarefied_analysis/qiime_2d-pcoa0/pcoa/pcoa_bray_curtis_otu_table_auto.txt: rarefied_analysis/qiime_2d-pcoa0/distance/bray_curtis_otu_table_auto.txt
	mkdir -p rarefied_analysis/qiime_2d-pcoa0/
	principal_coordinates.py -i rarefied_analysis/qiime_2d-pcoa0/distance/ -o rarefied_analysis/qiime_2d-pcoa0/pcoa/

rarefied_analysis/qiime_2d-pcoa0/pcoa_plots/_2D_PCoA_plots.html: rarefied_analysis/qiime_2d-pcoa0/pcoa/pcoa_bray_curtis_otu_table_auto.txt mapping/copy/metadata_mapping.txt
	mkdir -p rarefied_analysis/qiime_2d-pcoa0/
	make_2d_plots.py -i rarefied_analysis/qiime_2d-pcoa0/pcoa/ -m mapping/copy/metadata_mapping.txt -o rarefied_analysis/qiime_2d-pcoa0/pcoa_plots/

rarefied_analysis/qiime_compare-sample-groups0/distance/bray_curtis_otu_table_auto.txt: rarefy_otu_table/qiime_even0/otu_table_auto.biom phylogeny/qiime_fasttree/rep_set.tre
	mkdir -p rarefied_analysis/qiime_compare-sample-groups0/
	beta_diversity.py -i rarefy_otu_table/qiime_even0/otu_table_auto.biom -o rarefied_analysis/qiime_compare-sample-groups0/distance/ -m bray_curtis -t phylogeny/qiime_fasttree/rep_set.tre

rarefied_analysis/qiime_compare-sample-groups0/mrpp_results.txt: mapping/copy/metadata_mapping.txt rarefied_analysis/qiime_2d-pcoa0/distance/bray_curtis_otu_table_auto.txt
	mkdir -p rarefied_analysis/qiime_compare-sample-groups0/
	compare_categories.py --method mrpp -i rarefied_analysis/qiime_2d-pcoa0/distance/bray_curtis_otu_table_auto.txt -c Diet -o rarefied_analysis/qiime_compare-sample-groups0 -m mapping/copy/metadata_mapping.txt

rarefied_analysis/qiime_alpha-comparison0/alpha_rarefaction_plots/rarefaction_plots.html: create_otu_table/qiime0/otu_table.biom mapping/copy/metadata_mapping.txt phylogeny/qiime_fasttree/rep_set.tre
	mkdir -p rarefied_analysis/qiime_alpha-comparison0/
	alpha_rarefaction.py -f -i create_otu_table/qiime0/otu_table.biom -o rarefied_analysis/qiime_alpha-comparison0 -m mapping/copy/metadata_mapping.txt -t phylogeny/qiime_fasttree/rep_set.tre

rarefied_analysis/mesas_nmds0/nmds.pdf: convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab mapping/copy/metadata_mapping.txt
	mkdir -p rarefied_analysis/mesas_nmds0/
	mesas-nmds -i convert_otu_table/rarefied_biom_to_tab0/otu_table_auto.tab -m mapping/copy/metadata_mapping.txt -o rarefied_analysis/mesas_nmds0 -d bray

Feb26.zip: create_otu_table/qiime0 rarefy_otu_table/qiime_even0 convert_otu_table/full_biom_to_tab0 convert_otu_table/rarefied_biom_to_tab0 misc_analysis/biom_summarize-table0 misc_analysis/mesas_representative_sequences0 misc_analysis/mesas_representative_sequences0 misc_analysis/qiime_alpha_rarefaction0 rarefied_analysis/mesas_pcoa0 rarefied_analysis/mesas_indicator-species0 rarefied_analysis/mesas_mrpp0 rarefied_analysis/qiime_unifrac-pcoa0 rarefied_analysis/qiime_unifrac-pcoa0 rarefied_analysis/qiime_2d-pcoa0 rarefied_analysis/qiime_compare-sample-groups0 rarefied_analysis/qiime_alpha-comparison0 rarefied_analysis/mesas_nmds0 
	zip -r Feb26.zip report.html create_otu_table/qiime0 rarefy_otu_table/qiime_even0 convert_otu_table/full_biom_to_tab0 convert_otu_table/rarefied_biom_to_tab0 misc_analysis/biom_summarize-table0 misc_analysis/mesas_representative_sequences0 misc_analysis/mesas_representative_sequences0 misc_analysis/qiime_alpha_rarefaction0 rarefied_analysis/mesas_pcoa0 rarefied_analysis/mesas_indicator-species0 rarefied_analysis/mesas_mrpp0 rarefied_analysis/qiime_unifrac-pcoa0 rarefied_analysis/qiime_unifrac-pcoa0 rarefied_analysis/qiime_2d-pcoa0 rarefied_analysis/qiime_compare-sample-groups0 rarefied_analysis/qiime_alpha-comparison0 rarefied_analysis/mesas_nmds0 

.PHONY: all